app/controllers/concerns/member_helper.rb

```diff
@@ -59,7 +59,7 @@ def user_ids_for_new_members(member_params)
 
   def invite_new_users(user_ids, send_notification: true)
     user_ids.filter_map do |id|
-      if id.to_i == 0 && id.present? # we've got an email - invite that user
+      if id.present? && (id.to_i == 0 || EmailValidator.valid?(id)) # we've got an email - invite that user
```

Comment:
I believe this could be simplified:
```suggestion
      if EmailValidator.valid?(id) # we've got an email - invite that user
```

Reply:
Yes, but I guess the original implement is for performance reason as the logic is used in batch import also.

But I'm OK if member think the simplified version is OK.